test/jest.config.common.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
@kuzhelov not sure how much these things are related, but is this the place where we specify how the files in `test` directory are compiled? `ts-jest` vs `typescript` compiler? more details in #445